src/ignitus-StudentSignUp/Components/Interest.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, {Component} from 'react'
+import progress from '../Images/Group 12.2.png';
+import { Link } from 'react-router-dom';
+
+import { withErrorBoundary } from '../../ignitus-Internals';
+import group17 from '../Images/Group_17.png'
+
+import '../Styles/style.scss';
+
+class Interest extends Component {
+
+  render() {
+    return (
+      <div className="col-lg-12" style={{height: "80vw", backgroundColor: "white"}}>
+      <div className="progress-bar-signup"><img src={progress}></img></div>
+      <div className="welcome">What are you interested in?</div>
+      <div className="content1">This will customize your experience<br/>This will help you to discover right people and opportunities</div>
```

Comment:
please use semantic `classNames`, here `content1` is not signifying anything, or if `content1` is applicable to all paragraphs then use can use name like `className = "paragraph"`, and please avoid writing text as a **direct child** of `div` instead use `p` inside `div`.